module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -1037,34 +1242,50 @@ public function getHolds($refresh = true)
                 if (!empty($response)) {
                     $result->status = true;
                     $result->message = 'hold_place_success_html';
-                    $result->data = $response->holds;
-                    // Check for holds ready for chechout
-                    foreach ($response->holds as $key => $hold) {
-                        if (
-                            !$hold->autoCheckout
-                            && $hold->holdListPosition == 1
-                        ) {
-                            $result->data[$key]->holdReadyForCheckout = true;
-                            // format the expires date.
-                            $holdExpires = new \DateTime($hold->holdExpires);
-                            $result->data[$key]->holdExpires
-                                = $holdExpires->format(
+
+                    if (isset($response->holds)) {
+                        $result->data = [];
```

Comment:
You're assigning values to `$result->data` in two consecutive lines... since the empty array assigned in the first line is immediately overwritten, you don't need the first line at all.